src/components/search/Sort/sort.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../styles/utilities";
+
+.sort {
+  position: relative;
+  font-size: var(--text-size-2);
+  line-height: 1.25;
+
+  label {
+    margin-right: var(--space-1);
+    color: var(--color-text-subtle);
+  }
+
+  [data-store-select] {
+    min-height: var(--min-size);
+    padding: var(--space-1) var(--space-5) var(--space-1) var(--space-2);
+    color: var(--color-link);
+    background: transparent;
+    border: 0;
+    border-radius: var(--border-radius-default);
+    transition: background-color .2s ease;
+    appearance: none;
+
+    &:hover:not(:disabled) { background-color: var(--bg-secondary-light); }
+
+    &:disabled {
+      color: var(--color-text-disabled);
+      cursor: not-allowed;
+    }
+
+    &:focus {
+      @include focus-ring;
+    }
+  }
+
+  svg {
+    position: absolute;
+    top: calc(50% - 9px);
```

Comment:
Hmm. I'm not sure because the arrow is supposed to be inside the `select` element, isn't it?

![image](https://user-images.githubusercontent.com/3356699/149016009-2fb18c74-4ecd-4b02-954a-ef1aef0e8b42.png)

Reply:
never mind. Got it!